static/src/javascripts/projects/commercial/modules/messenger/background.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+import { addEventListener } from 'lib/events';
+import fastdom from 'lib/fastdom-promise';
+
+import { register } from 'commercial/modules/messenger';
+
+type AdSpec = {
+    scrollType: string,
+    backgroundColor: string,
+    backgroundImage: string,
+    backgroundRepeat: string,
+    backgroundPosition: string,
+};
+
+const setBackground = (specs: AdSpec, adSlot: Node): Promise<any> | null => {
+    if (
+        !specs ||
+        !('backgroundImage' in specs) ||
+        !('backgroundRepeat' in specs) ||
+        !('backgroundPosition' in specs) ||
+        !('scrollType' in specs)
+    ) {
+        return null;
+    }
+
+    const specsStyles = Object.keys(specs).reduce((result, key) => {
+        if (key !== 'scrollType') {
+            result[key] = specs[key];
+        }
+        // Flow is love, Flow is Life! DFP has been passing us `backgroundColour`
+        // all along, and the setting of this css prop has been silently failing
+        if (key === 'backgroundColour') {
+            result.backgroundColor = specs[key];
+        }
+        return result;
+    }, {});
+
+    // Create an element to hold the background image
+    const background = document.createElement('div');
+    background.className = `creative__background creative__background--${specs.scrollType}`;
+    background.style = Object.assign(background.style, specsStyles);
```

Comment:
Goodbye lodash-assign, hello `Object.assign`

Because Flow is smart enough to check through the [CSSStyleDeclaration](https://github.com/facebook/flow/blob/v0.46.0/lib/cssom.js#L74): 
1) we can no longer overload the styles and set a style for `scrollType`, so we must filter it out
2) backgroundColour !== backgroundColor 😅 so we should add another key, with a name CSS will recognise, in case any creative wants to actually set the `background-color`

To stop Flow complaining because of these things, I'm making a new object called specStyles on line 26

##### Before:
background.style = {0: "background-image", 1: "background-repeat-x", 2: "background-repeat-y", 3: "background-position-x", 4: "background-position-y", **scrollType: "fixed"**, backgroundColour: "#ebebeb", alignContent: "", alignItems: "", alignSelf: ""…}

##### After:
background.style = {0: "background-image", 1: "background-repeat-x", 2: "background-repeat-y", 3: "background-position-x", 4: "background-position-y", backgroundColour: "#ebebeb", **backgroundColor: "#ebebeb"**, alignContent: "", alignItems: "", alignSelf: ""…}
 
backgroundColour is still there, not sure if it is worth the overhead to delete it... if it does nothing anyway